### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by R1meq**

### CODE REVIEW

File: Land.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	for (const auto& country : listCountries)
+	{
+		country.displayinfo();
+		cout << endl;
+		if (--topCount == 0)
+			break;
+	}
+}
+
+Country Land::TopByPopulation(governmentType type) const
```
Comment (https999) [inline Land.cpp]: The first character of function/ method argument names should be lowercase. All words starting after the first letter should be in the upper case with class names.

File: Land.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	list<Country> listCountries;
+public:
+	Land();
+	Land(string name, list<Country> listCountries);
+	void setName(string name);
+	string getName() const;
+	void sortByGPD();
+	void printLand();
+	void printLandTopByGPD(const int &count);
+	Country TopByPopulation(governmentType type) const;
```
Comment (https999) [inline Land.h:20]: назва метода має бути  topByPopulation
перше слово по конвеншину для методів має бути з малої буки  ,а не з великої

**Review by https999 — CHANGES_REQUESTED** [review]

File: main.cpp [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	Land land("Europe", countries);
+
+	land.printLand();
+	cout << "PopulationDensity: " << сalculatePopulationDensity(countries) << " million/km" << endl;
+	cout << "--------------------- Output after sort by GPD-----------------------" << endl;
+	land.sortByGPD();
+	land.printLand();
+	cout << endl;
+	cout << "--------------------- Output Top 3 Country by GPD-----------------------" << endl;
+	const int topCount = 3;
```
Comment (https999) [inline main.cpp]: Variables declared constexpr or const, and whose value is fixed for the duration of the program, are named with a leading "k" followed by mixed case.

File: Land.cpp [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+void Land::sortByGPD()
+{
+	listCountries.sort(compCountryByGPDByDecrease);
+}
+
+void Land::printLandTopByGPD(const int &count)
+{
+	int topCount = count;
```
Comment (https999) [inline Land.cpp]: variable name lowercase with underscore  between words

File: Country.cpp [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+	cout << "population: " << population<< " millions" << endl;
+	cout << "area: " << area <<" km"<< endl;
+	cout << "GPD: " <<"$" << GPD << " Per capita" << endl;
+	cout << "code: " <<"+"<< code << endl;
+	cout << "government type: " << governmentTypeToString.find(type)->second << endl;
+}
+
+double сalculatePopulationDensity(list<Country>& Countries)
+{
+	double sumPopulation = 0.0;
```
Comment (https999) [inline Country.cpp]: variable name lowercase with underscore  between words

File: Country.cpp [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+	double sumArea = 0.0;
```
Comment (https999) [inline Country.cpp]: variable name lowercase with underscore  between words

**Review by https999 — APPROVED** [review]

**Comment by https999** [comment]
Тhe name of the commit messages have been fixed to be understandable

**Comment by R1meq** [comment]
Тhe name of the commit messages have been fixed to be understandable

**Comment by https999** [comment]
Great job!

**Pull request closed without merging**